addons/l10n_cl/models/account_move.py

```diff
@@ -124,3 +124,19 @@ def _get_name_invoice_report(self):
         if self.l10n_latam_use_documents and self.company_id.country_id.code == 'CL':
             return 'l10n_cl.report_invoice_document'
         return super()._get_name_invoice_report()
+
+    @api.onchange('l10n_latam_document_type_id', 'l10n_latam_document_number')
+    def _inverse_l10n_latam_document_number(self):
```

Comment:
After a rebase it should work without overriding this method. @suv-bmya

Reply:
👍